qutebrowser/mainwindow/statusbar/tabindex.py

```diff
@@ -0,0 +1,42 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""TabIndex displayed in the statusbar."""
+
+from PyQt5.QtCore import pyqtSlot
+
+from qutebrowser.mainwindow.statusbar import textbase
+from qutebrowser.utils import objreg
+
+class TabIndex(textbase.TextBase):
+
+    """Shows current tab index and number of tabs in the statusbar."""
+
+    def __init__(self, win_id, parent=None):
+        super().__init__(parent)
+        self.tabbed_browser = objreg.get('tabbed-browser',
+                                         scope = 'window',
+                                         window = win_id)
+
+    @pyqtSlot(int)
+    def on_tab_changed(self, tab):
+        """Update tab index when tab changed."""
+        self.setText('[{0}/{1}]'.format(
```

Comment:
Please simply use `[{}/{}]` here - python3 doesn't need the explicit numbering :)